### PR TITLE
Added ice axe and allowed it to spawn in sport shops

### PR DESCRIPTION
**Pull request by Night-Pryanik**

### CODE REVIEW

File: data/json/items/tools.json [hunk 1]
```diff
@@ -10797,5 +10797,30 @@
...
+    "color": "dark_gray",
+    "name": "ice axe",
+    "description": "This is an ice axe with hammer on its head, a multi-purpose hiking and climbing tool used by mountaineers.  It is sturdy enough, so you can use it to pry open closed doors or lift manhole covers.",
+    "price": 5000,
+    "material": "steel",
+    "weight": 500,
+    "volume": 3,
+    "bashing": 14,
+    "cutting": 3,
+    "flags" : ["STAB", "DURABLE_MELEE"],
```
Comment (Rivet-the-Zombie) [inline data/json/items/tools.json]: I'd say this is more of a ````"CHOP"```` weapon than a ````"STAB"```` one.

Comment (Night-Pryanik) [inline data/json/items/tools.json]: Well, it have a narrow, pointy end, like the one a pickaxe have.
371px-icetool
It doesn't have cutting edge to chop.

Comment (Rivet-the-Zombie) [inline data/json/items/tools.json]: The STAB flag specifically gives messages like 'You stab the zombie with your ice axe!' and reduces stick-in chance by a great deal. That thing has a serrated blade; it's going to stick.

Comment (Night-Pryanik) [inline data/json/items/tools.json]: I had in mind this information from JSON_FLAGS.md: "```STAB``` Deals stabbing damage, with a moderate chance of getting stuck."
That's just what I wanted - have a stabbing damage and a chance to stick.

Comment (Rivet-the-Zombie) [inline data/json/items/tools.json]: Half the normal chance, plus the combat messages.

Comment (Night-Pryanik) [inline data/json/items/tools.json]: Well, ```chop``` flag isn't appropriate for this tool, ```stab``` flag gives too small chance of sticking. If I decide not to put any of this flags, then tool will get "cutting" damage, right? This isn't appropriate too.
I think ```stab``` is a the lesser of three evils.

**Pull request merged**